omniscidb/QueryEngine/ColumnFetcher.cpp

```diff
@@ -478,6 +481,18 @@ const int8_t* ColumnFetcher::linearizeColumnFragments(
   // prepare ChunkIter for the linearized chunk
   auto merged_chunk = std::make_shared<Chunk_NS::Chunk>(
       merged_data_buffer, merged_index_buffer, col_info);
+  merged_data_buffer->deleteWhenUnpinned();
```

Comment:
It's enough to call this method once (in dtor).

Reply:
done, review again, please